lib/tests/runner.rs

```diff
@@ -0,0 +1,28 @@
+#[path = "../../cli/tests/test_no_abandoned_test_files.rs"]
+mod test_no_abandoned_test_files;
```

Comment:
nit: this looks a bit cryptic.

Perhaps, the inner function can be extracted to `testutils`? Then we can add a thin `#[test]` that calls the function with the runner file path?

Reply:
great idea, done! that also lets me move the `#[test]` to `runner.rs` as @ilyagr suggested :)